tools/checkCommitMsg.ts

```diff
@@ -0,0 +1,39 @@
+import * as fs from 'fs';
```

Comment:
Following Nx guidelines, this should be under a `scripts` subfolder: `tools/scripts/checkCommitMsg.ts`